### PR TITLE
Removing unused chart

### PR DESCRIPTION
**Pull request by bCatanant**
Removing unused chart on charles C.D.

### CODE REVIEW

**Review by arthurpuccizup — CHANGES_REQUESTED** [review]
Why not remove the property health in the datasource registration, if there is no more health, this flag is not necessary anymore

**Review by maycommit — DISMISSED** [review]

**Comment by icaroafonsozup** [comment]
> Why not remove the property health in the datasource registration, if there is no more health, this flag is not necessary anymore

Property and references removed as asked.

File: ui/src/modules/Circles/Menu/index.tsx [hunk 1]
```diff
@@ -61,10 +60,7 @@ const CirclesFilter = ({ status, onSearch, onSelect, children }: Props) => {
           </LabeledIcon>
         </Menu>
         <Styled.A onClick={() => goToCircleComparison(CIRCLE_MATCHER_TAB)}>
-          <Styled.Icon name="circle-matcher" color="dark" size="15px" />
-        </Styled.A>
-        <Styled.A as={LinkRouter} to={routes.circlesMetrics}>
-          <Styled.Icon name="activity" color="dark" />
+          <Styled.Icon name="circle-matcher" color="dark" size="15px" title={'circle matcher'} />
```
Comment (ifdouglas) [inline ui/src/modules/Circles/Menu/index.tsx:63]:
```suggestion
          <Styled.Icon name="circle-matcher" color="dark" size="15px" title="circle matcher" />
```

**Review by ifdouglas — APPROVED** [review]

**Review by ifdouglas — DISMISSED** [review]

File: compass/internal/tests/datasource_test.go [hunk 2]
```diff
@@ -4,7 +4,8 @@
  *
  *  Licensed under the Apache License, Version 2.0 (the "License");
  *  you may not use this file except in compliance with the License.
- *  You may obtain a copy of the License at
+ *  You may obtain a copy of the License atFindAllByWorkspace
```
Comment (arthurpuccizup) [inline compass/internal/tests/datasource_test.go:7]: Miss typing in the license

File: compass/migrations/20210303120000_alter_data_sources_table.down.sql [hunk 3]
```diff
@@ -0,0 +1 @@
+ALTER TABLE data_sources DROP COLUMN HEALTH;
```
Comment (arthurpuccizup) [inline compass/migrations/20210303120000_alter_data_sources_table.down.sql:1]: This is not the "down" script, this is the "up" script, which by the way is missing

**Review by arthurpuccizup — CHANGES_REQUESTED** [review]

File: compass/internal/tests/datasource_test.go (hunk 2 above)
Comment (icaroafonsozup) [reply compass/internal/tests/datasource_test.go]: Resolved as asked.

File: compass/migrations/20210303120000_alter_data_sources_table.down.sql (hunk 3 above)
Comment (icaroafonsozup) [reply compass/migrations/20210303120000_alter_data_sources_table.down.sql]: Resolved as asked.

**Review by arthurpuccizup — APPROVED** [review]

**Review by ifdouglas — APPROVED** [review]

**Pull request merged**